### PR TITLE
added fields/{id} GET method (feature/#129)

### LINKED ISSUE #129: Implement Field/{id} resource GET

### PR DESCRIPTION
**Pull request by Howrran**

### CODE REVIEW

**Review by IvanKostiv — DISMISSED** [review]

File: src/app/routers/field.py [hunk 1]
```diff
@@ -170,3 +173,45 @@ def get(self):
...
+        """
+        Get field by id
+
+        :param field_id: field id
+        :return: json
+        """
+        field = FieldService.get_by_id(field_id)
+
+        if field is None:
+            raise BadRequest("Field does not exist")
```
Comment (yurdosii) [inline src/app/routers/field.py:206]: BadRequest is 400 response. No 400 response in API.doc

Comment (Howrran) [reply src/app/routers/field.py:206]: added

**Review by IvanKostiv — DISMISSED** [review]

File: src/app/routers/field.py [hunk 2]
```diff
@@ -170,3 +173,45 @@ def get(self):
...
+
+        url: '/fields/{id}'
+        methods: GET, PUT, DELETE
+    """
+
+    @API.doc(
+        responses={
+            200: 'OK',
+            403: 'User is not the field owner',
+            404: 'Field not found',
```
Comment (IvanKostiv) [inline src/app/routers/field.py]: Not added 400 code

File: src/app/routers/field.py [hunk 3]
```diff
@@ -170,3 +173,46 @@ def get(self):
...
+        url: '/fields/{id}'
+        methods: GET, PUT, DELETE
+    """
+
+    @API.doc(
+        responses={
+            200: 'OK',
+            400: 'Bad Request',
+            403: 'User is not the field owner',
+            404: 'Field not found',
```
Comment (yurdosii) [inline src/app/routers/field.py]: Where do you raise 404 ?

**Review by yurdosii — APPROVED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Pull request merged**